src/cli/Container.ts

```diff
@@ -246,6 +258,21 @@ export class Container {
     return this.getDownloadIdmsFileCommand(idmsGroupFilename);
   }
 
+  public async getLambda(): Promise<AWS.Lambda> {
+    const region = process.env.LAMBDA_REGION || 'eu-west-1';
```

Comment:
Theoretically it should work without credentials here because I added IAM Role to ECS Odyssey definition.

We'll see :-) I bet with AZ that it won't work.

Reply:
if role is configured correctly it will work :)